packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -320,31 +334,95 @@ function WorklistDataGrid(props) {
   );
 
   const handleClearRowSelection = () => {
+    let message = '';
+    rowSelection.current.lastIndividuallySelectedRow = null;
     if (isAnyRowSelected()) {
-      setAriaLiveMessage(intl.formatMessage({ id: 'Terra.worklist-data-grid.all-rows-unselected' }));
+      message = intl.formatMessage({ id: 'Terra.worklist-data-grid.all-rows-unselected' });
```

Comment:
What required this change?

Reply:
The merge conflict initiated this change.